src/css/jwplayer/imports/video.less

```diff
@@ -17,25 +17,25 @@
     }
 
     &.jw-stretch-uniform {
-        video {
+        .jw-media video {
             object-fit: contain;
         }
     }
 
     &.jw-stretch-none {
-        video {
+        .jw-media video {
             object-fit: none;
         }
     }
 
     &.jw-stretch-fill {
-        video {
+        .jw-media video {
             object-fit: cover;
         }
     }
 
     &.jw-stretch-exactfit {
-        video {
+        .jw-media video {
```

Comment:
The extra specificity prevents these styles from being applied to motion thumbnail video tags?

Reply:
Yep.